verilog/analysis/default_rules.h

```diff
@@ -65,6 +65,7 @@ constexpr const char* kDefaultRuleSet[] = {
     "positive-meaning-parameter-name",
     "constraint-name-style",
     "suggest-parentheses",
+    "truncated-numeric-literal",
```

Comment:
I used to recommend flipping the default switch separately to make it easily revertible.

Reply:
sounds good, will comment here and make separate PR with just that flip.